File: classificador.java
public class HeroiClassificador {
    public static void main(String[] args) {
        String nome = "Aragorn";
        int xp = 2320;
        String nivel;

        if (xp < 1000) {
            nivel = "Ferro";
        } else if (xp <= 2000) {
            nivel = "Bronze";
        } else if (xp <= 5000) {
            nivel = "Prata";
        } else if (xp <= 7000) {
            nivel = "Ouro";
        } else if (xp <= 8000) {
            nivel = "Platina";
        } else if (xp <= 9000) {
            nivel = "Ascendente";
        } else if (xp <= 10000) {
            nivel = "Imortal";
        } else {
            nivel = "Radiante";
        }

        System.out.println("O Herói de nome " + nome + " está no nível de " + nivel);
    }
}

File: classificador.rb
nome = "Aragorn"
xp = 2320

nivel = if xp < 1000
          "Ferro"
        elsif xp <= 2000
          "Bronze"
        elsif xp <= 5000
          "Prata"
        elsif xp <= 7000
          "Ouro"
        elsif xp <= 8000
          "Platina"
        elsif xp <= 9000
          "Ascendente"
        elsif xp <= 10000
          "Imortal"
        else
          "Radiante"
        end

puts "O Herói de nome #{nome} está no nível de #{nivel}"
